src/libraries/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.Experimental.cs

```diff
@@ -230,6 +230,56 @@ public abstract partial class AdvSimd : System.Runtime.Intrinsics.Arm.ArmBase
         public unsafe static System.Runtime.Intrinsics.Vector64<float> LoadVector64(float* address) { throw null; }
         public unsafe static System.Runtime.Intrinsics.Vector64<ushort> LoadVector64(ushort* address) { throw null; }
         public unsafe static System.Runtime.Intrinsics.Vector64<uint> LoadVector64(uint* address) { throw null; }
+        public static System.Runtime.Intrinsics.Vector128<byte> Max(System.Runtime.Intrinsics.Vector128<byte> left, System.Runtime.Intrinsics.Vector128<byte> right) { throw null; }
```

Comment:
@echesakovMSFT same comment as in the other PR (although since this is an Experimental file, you can probably ignore the request): We have a new rule that new public APIs need to be documented with triple slash in their implementation. Please make sure to do that. Or since it's experimental, make sure to get it documented once they become official.

Reply:
This is the ref file and they are not normally documented.

We have been added the relevant `///` comments to the added APIs in S.P.Corelib.

Reply:
Yes, ref files should not be documented. Thanks for making sure the API definitions are being documented.